compiler/noirc_frontend/src/tests.rs

```diff
@@ -902,6 +902,31 @@ mod test {
         assert!(get_program_errors(src).is_empty());
     }
 
+    #[test]
+    fn resolve_literal_slice() {
+        let src = r#"
+            fn const_0(_x : [Field]) -> Field {
+                0
+            }
+            fn main(x : Field) {
+                let y: [Field] = &[1, 2, 3];
+                assert(const_0(y) == x);
+            }
+        "#;
+        assert!(get_program_errors(src).is_empty());
+    }
+
+    #[test]
+    fn index_literal_slice() {
+        let src = r#"
+            fn main(x : Field) {
+                let y: [Field] = &[1, 2, 3];
+                assert(y[0] == x);
+            }
+        "#;
+        assert!(get_program_errors(src).is_empty());
+    }
```

Comment:
Wouldn't these cases already be tested for in the `slices` execution test (and others)?
I think if slices were ever to not resolve or index correctly it'd be relatively easy to narrow down a larger example to this case if needed.

Reply:
yes, this was particularly useful when debugging errors in the stdlib, but I've removed it.